src/components/molecules/Header.tsx

```diff
@@ -56,9 +56,11 @@ const Header: FC = () => {
     <AppBar>
       <Logo src={AnywayImage} alt={'Anyway'} height={30} onClick={reloadHomePage} />
       <Box className={classes.userSection}>
-        <LanguageMenu />
-        {authElement && (
+        {FEATURE_FLAGS.login ? (
```

Comment:
1. why use `FEATURE_FLAGS.login` twice? if the flag is `false` you can leave `authElement` as null and use it for the condition
2. ` <>{authElement}</>` is the same `authElement`